interpreter/interpreter.go

```diff
@@ -4600,7 +4613,7 @@ func authAccountReadFunction(
 	)
 }
 
-func authAccountBorrowFunction(
+func authAccountStorageBorrowFunction(
```

Comment:
This was refactored to extract out the common-code. No change to the business logic was done.